code/datums/components/storage/storage.dm

```diff
@@ -232,13 +232,16 @@
 		stoplag(1)
 	qdel(progress)
 
-/datum/component/storage/proc/mass_remove_from_storage(atom/target, list/things, datum/progressbar/progress)
+/datum/component/storage/proc/mass_remove_from_storage(atom/target, list/things, datum/progressbar/progress, trigger_on_found = TRUE)

```

Comment:
make sure to add this variable to any overrides of the proc

Reply:
think this is the only one